photutils/detection/findstars.py

```diff
@@ -7,18 +7,20 @@
 import warnings
 import math
 import numpy as np
+import abc
 from astropy.table import Column, Table
 from astropy.utils.exceptions import AstropyUserWarning
 from astropy.stats import gaussian_fwhm_to_sigma
 from .core import _convolve_data, find_peaks
 
 
-__all__ = ['daofind', 'irafstarfind']
```

Comment:
You don't want to get rid of this, because it's what tells the rest of the package what's "in" this module. Instead you'll want to change it to match the new names - e.g., ``__all__ = ['StarFinder', 'DAOFind', ''IRAFStarFind]